lib/matplotlib/widgets.py

```diff
@@ -1135,16 +1135,21 @@ def _update(self):
 
 class _SelectorWidget(AxesWidget):
 
-    def __init__(self, ax, onselect, useblit=False, button=None):
+    def __init__(self, ax, onselect, useblit=False, button=None,
+                 state_modifier_keys=None):
         AxesWidget.__init__(self, ax)
 
         self.visible = True
         self.onselect = onselect
+        self.useblit = useblit and self.canvas.supports_blit
         self.connect_default_events()
 
+        self.state_modifier_keys = dict(move=' ', clear='escape',
+                                        square='shift', center='ctrl')
```

Comment:
I like this. It also helps to keep all the possible actions and associated keys in one place.